linux_os/guide/system/accounts/accounts-session/file_permissions_home_dirs/tests/correct_permissions.pass.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+#
+
+useradd -m cacusertest
+chmod -f u-s,g-w-s,o=- /home/*
```

Comment:
Hmm, test scenarios for `file_permissions_home_dirs` are failing when I run them with `cis` profile. I have checked the built RHEL8 data stream and the rule `file_permissions_home_dirs` is not selected in the `cis` profile even though it is listed here: https://github.com/ComplianceAsCode/content/blob/master/controls/cis_rhel8.yml#L2203
So this is not an issue in the test scenarios, it seems our controls are not working properly?

Reply:
@yuumasato @jan-cerny @matejak  ^^ I've used the `build_product` script to build the RHEL8 data stream.

Reply:
@matusmarhefka @marcusburghardt The `related_rules` are not included in the Profile.

Reply:
It's not a bug, it's a feature :smiley:

Reply:
@yuumasato @jan-cerny Hmm, the rule should cover CIS benchmark's recommendation `6.2.7` so I don't understand why it is not included in the profile. Can you explain?

Reply:
Ohh, now I see: `The rule below exists, but does not have any OVAL checks or remediations.`. But the rule already has the OVAL check.

Reply:
Anyway, inconsistency with the `file_permissions_home_dirs` rule in our controls is not related to this PR so I am going to merge it.